README.md

```diff
@@ -33,21 +33,21 @@ This sample walks you through the process of integrating authentication with Mic
 ## Scenario
 
 This sample app is a [multi-tenant](https://docs.microsoft.com/en-us/azure/active-directory/develop/setup-multi-tenant-app) app, which means that it can sign-in users from any Azure AD tenant and Microsoft Accounts.  It also demonstrates how a developer can build apps to connect with enterprise users and access their Azure + O365 data via [Microsoft Graph](https://docs.microsoft.com/en-us/graph/overview).
-During the auth flow, the users will be required to sign-in first, if its their first time signing-in to the app, the user would be asked to consent to the [permissions](https://docs.microsoft.com/en-us/azure/active-directory/develop/v1-permissions-and-consent) required by the application.
+During the auth flow, the users will be required to sign-in first, if it is their first time signing-in to the app, the user would be asked to consent to the [permissions](https://docs.microsoft.com/en-us/azure/active-directory/develop/v1-permissions-and-consent) required by the application.
 
 The majority of the logic in this sample shows how to sign-in an end user and make a call to the Microsoft Graph to get basic information about the signed-in user.
 
 ![Flowchart](ReadmeFiles/image1.png)
 
 ## Broker Authentication using MSAL
 
-Microsoft provides applications for every mobile platform that allow for the bridging of credentials across applications from different vendors and for enhanced features that require a single secure place from where to validate credentials. These are called brokers. The brokers available for Android are **Microsoft Authenticator** and **Company Portal**.
-[Learn more about Brokers here.](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-v1-enable-sso-android#single-sign-on-concepts)
+Microsoft provides applications for every mobile platform that allow for the bridging credentials across applications from different vendors and for enhanced features that require a single secure place from where to validate credentials. These applications are called Brokers. The brokers available for Android are **Microsoft Authenticator** and **Company Portal**.
+[Learn more about Brokers here.](https://docs.microsoft.com/azure/active-directory/develop/brokered-auth)
 
 The MSAL for Android will automatically use the broker if they are present on the device.
 
-> [!NOTE]
-> If you have older versions of **Microsoft Authenticator** app or [Company portal app](https://docs.microsoft.com/en-us/intune-user-help/install-and-sign-in-to-the-intune-company-portal-app-ios) installed in the device where this sample application will be run, then the user might not be able to test the scenarios presented here. Please make sure that you have installed the latest version of Microsoft Authenticator or Company Portal on your device.
+> **Note:**

```

Comment:
Related to [this question](https://github.com/Azure-Samples/ms-identity-android-java/pull/10/files#r329155281): if this _is_ going to be rendered on the docs site, the `[!NOTE]` blocks get special CSS styling.

Reply:
Nope. Content from readme is not getting picked up by docs.